go.mod

```diff
@@ -1,42 +1,261 @@
 module github.com/libopenstorage/operator
 
-go 1.15
+go 1.19
 
 require (
 	github.com/dgrijalva/jwt-go v3.2.1-0.20180719211823-0b96aaa70776+incompatible
-	github.com/go-logr/logr v1.1.0
+	github.com/go-logr/logr v1.2.3
 	github.com/golang/mock v1.6.0
+	github.com/golangci/golangci-lint v1.49.0
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/google/uuid v1.3.0
-	github.com/hashicorp/go-version v1.2.1
+	github.com/hashicorp/go-version v1.6.0
 	github.com/libopenstorage/cloudops v0.0.0-20200604165016-9cc0977d745e
-	github.com/libopenstorage/openstorage v8.0.1-0.20211105030910-665c2f474186+incompatible
+	github.com/libopenstorage/openstorage v9.4.20+incompatible
 	github.com/openshift/api v0.0.0-20210105115604-44119421ec6b
 	github.com/portworx/kvdb v0.0.0-20200723230726-2734b7f40194
-	github.com/portworx/sched-ops v1.20.4-rc1.0.20220725231657-5a6a43c6a5b3
+	github.com/portworx/sched-ops v1.20.4-rc1.0.20220927173018-c9d3edcaef50
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.46.0
 	github.com/prometheus-operator/prometheus-operator/pkg/client v0.46.0
-	github.com/sirupsen/logrus v1.8.1
-	github.com/stretchr/testify v1.7.0
+	github.com/sirupsen/logrus v1.9.0
+	github.com/stretchr/testify v1.8.0
 	github.com/urfave/cli v1.22.2
-	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616 // indirect
-	golang.org/x/sys v0.0.0-20220818161305-2296e01440c6 // indirect
-	golang.org/x/tools v0.1.12 // indirect
-	golang.org/x/xerrors v0.0.0-20220411194840-2f41105eb62f // indirect
-	google.golang.org/grpc v1.40.0
+	google.golang.org/grpc v1.47.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.21.4
-	k8s.io/apiextensions-apiserver v0.21.4
-	k8s.io/apimachinery v0.21.4
+	k8s.io/api v0.25.1
+	k8s.io/apiextensions-apiserver v0.25.1
+	k8s.io/apimachinery v0.25.1
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/code-generator v0.21.4
-	k8s.io/component-helpers v0.21.4
+	k8s.io/code-generator v0.25.1
+	k8s.io/component-helpers v0.25.1
 	k8s.io/klog v1.0.0
-	k8s.io/kubernetes v1.20.11
-	k8s.io/utils v0.0.0-20210802155522-efc7438f0176
+	k8s.io/kubernetes v1.25.1
+	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
 	sigs.k8s.io/cluster-api v0.2.11
 	sigs.k8s.io/controller-runtime v0.9.7
```

Comment:
not upgrading the controller-runtime as well?

Reply:
Actually that was a good idea. I was planning to do it when I add the controller-tools for automatic CRD schema generation. I will add all of that in a separate PR.

Reply:
last time when I do the upgrade to 1.21.4, I remember I didn't upgrade it to the latest version, because I had a sense we need to keep it as low as possible.. so I picked the lowest compatible version. I don't know if it's good to upgrade to the latest version.